### ISSUE TITLE
Instant.fromEpochMilliseconds fails on Kotlin/JS

### ISSUE THREAD

**Issue by Sternbach-Software** [Kotlin/kotlinx-datetime#298]
```
val x = runCatching { dynamicVariable.unsafeCast<Long>() }.getOrNull()
if(x == null) Clock.System.now() else Instant.fromEpochMilliseconds(x)
```
Throws:
```
long.kt:145 Uncaught TypeError: epochMilliseconds.div_9s1fi3_k$ is not a function
    at protoOf.fromEpochMilliseconds_tgp1r9_k$ (long.kt:145:56)
    at myFunction (MyApp.js.kt:32:93)
    at l (MyApp.js.kt:16:44)
```

**Comment by ilya-g** [comment]
Are you sure that the value of dynamicVariable is really Long? Try using `as Long` instead of `unsafeCast` function and see if it fails on casting

**Comment by Sternbach-Software** [comment]
That is not what is crashing.

**Comment by ilya-g** [comment]
I could not reproduce the problem with the latest released version 0.4.0 on the following code:
```
    @Test
    fun fromEpochMillisDynamic() {
        val millis: dynamic = 1_987_999_233_000L
        val x = runCatching { millis.unsafeCast<Long>() }.getOrNull()
        val instant = if (x == null) Clock.System.now() else Instant.fromEpochMilliseconds(x)
        println(instant) // 2032-12-30T06:00:33Z
    }
```

Please post a self-contained sample that exhibits this problem.

**Comment by Sternbach-Software** [comment]
The `else` never executes. I was running into this error in a previous commit of https://GitHub.com/SternbachSoftware/BeautifulZmanim. I will find the commit.
